libs/libc/time/lib_localtime.c

```diff
@@ -363,9 +365,9 @@ FAR char *tzname[2] =
 
 static int_fast32_t detzcode(FAR const char *codep);
 static int_fast64_t detzcode64(FAR const char *codep);
-static int  differ_by_repeat(time_t t1, time_t t0);
+static int differ_by_repeat(time_t t1, time_t t0);
 static FAR const char *getzname(FAR const char *strp);
-static FAR const char *getqzname(FAR const char *strp, const int delim);
+static FAR const char *getqzname(FAR const char *strp, int delim);
```

Comment:
@xiaoxiang781216 Why are you dropping ALL the const specifiers?

Reply:
The const modifier for non pointer argument isn't very useful, and all other NuttX functions never add const to the non pointer argument even the argument is never changed in the function body. This change make the locale time more consistent with others.